.github/workflows/ci-merge-checks.yml

```diff
@@ -1,6 +1,5 @@
 on:
   pull_request:
-    types: [ready_for_review]
```

Comment:
We can restore this to its original state. It was used to force tests from the `epic/a11y` branch.